ldt/src/ldt_based_pcs.rs

```diff
@@ -250,11 +276,3 @@ where
             .verify(&quotient_mmcs, &dims, &commits, proof, challenger)
     }
 }
-
-fn transpose<T: Clone>(vec: Vec<Vec<T>>) -> Vec<Vec<T>> {
```

Comment:
I think we can safely remove this code. Happy to added it back, if necessary.

Reply:
SGTM, let's remove for now